iac-extensions/docker/src/test/java/org/sonar/iac/docker/plugin/DockerRulesDefinitionTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.docker.plugin;
+
+import org.junit.jupiter.api.Test;
+import org.sonar.api.SonarEdition;
+import org.sonar.api.SonarQubeSide;
+import org.sonar.api.SonarRuntime;
+import org.sonar.api.internal.SonarRuntimeImpl;
+import org.sonar.api.server.rule.RulesDefinition;
+import org.sonar.api.utils.Version;
+import org.sonar.iac.docker.checks.DockerCheckList;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class DockerRulesDefinitionTest {
+
+  @Test
+  void testActivationSonarLint() {
+    RulesDefinition.Repository repository = dockerRuleRepository(9, 3);
+    assertThat(repository).isNotNull();
```

Comment:
Hint, minor. There is no need to call `isNotNull()` since in the next line is `isEqualTo` check which is more precise (and also check is not null)